pkg/stack/unwind/compact_unwind_table_test.go

```diff
@@ -455,8 +456,10 @@ func TestIsSorted(t *testing.T) {
 	matches, err := filepath.Glob("../../../testdata/vendored/x86/*")
 	require.NoError(t, err)
 
+	logger := logger.NewLogger("error", logger.LogFormatLogfmt, "compact-unwind-table-tests")
```

Comment:
You're misusing the API here. What you need is `log.With`. You can find a lot of examples in the codebase.

https://github.com/go-kit/log?tab=readme-ov-file#contextual-loggers

Reply:
TIL about contextual logging, thanks!🤩
Have updated the PR